package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@gurukumparan/react-native-android-inapp-updates",
   "title": "React Native Android Inapp Updates",
-  "version": "1.0.21",
```

Comment:
Hi @HorbachAndrii thank you for your awesome pr, can you revert the versioning to default?. @kumparan/react-native will handle the deployment and versioning to NPM. cc @renopp

Reply:
Roger that)

Reply:
Thank you for the swift response @HorbachAndrii.